src/components/sections/columns.json

```diff
@@ -0,0 +1,49 @@
+{
+  "collectionName": "components_sections_columns",
+  "info": {
+    "displayName": "columns",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "topWave": {
```

Comment:
The rules of the wave should be clarified because I'm not sure it should be from the top and the bottom. Also, I'm not sure that color is needed for wave.

Reply:
Thank you, will try to explain my vision in the meet-up